examples/slave/nuttx/xmc4800/EEPROM_factory.cc

```diff
@@ -0,0 +1,40 @@
+#include "EEPROM_factory.h"
+
+#include <cstring>
+
+namespace kickcat
+{
+    void createEEPROM(eeprom_description& eeprom, EEPROM_default type)
```

Comment:
unused type?

Reply:
Could be a switch case based on the type as an embryo of factory (since we don't have other conf at the moment)